libs/community/langchain_community/chat_models/tongyi.py

```diff
@@ -729,6 +731,7 @@ def _stream(
             if (
                 choice["finish_reason"] == "null"
                 and message["content"] == ""
+                and message["reasoning_content"] == ""
```

Comment:
It seems that not all models have `"reasoning_content"`, if the model is not qwq, it will raise an error:
```
    |   File "lib/python3.11/site-packages/langchain_core/language_models/chat_models.py", line 896, in _generate_with_cache
    |     for chunk in self._stream(messages, stop=stop, **kwargs):
    |   File "lib/python3.11/site-packages/langchain_community/chat_models/tongyi.py", line 734, in _stream
    |     and message["reasoning_content"] == ""
    |         ~~~~~~~^^^^^^^^^^^^^^^^^^^^^
    | KeyError: 'reasoning_content'
```

The key should be checked if it is existed first here.

Reply:
Fixed in https://github.com/langchain-ai/langchain/pull/30694